gerrit_python_tools/gerrit.py

```diff
@@ -624,13 +682,30 @@ def send_upstream(self, downstream, upstream):
         approvals = self.get_approvals(downstream.SSH())
 
         # Check to see if comment has necessary approvals.
-        if not self.is_upstream_approved(approvals):
-            msg = ("Could not send to upstream: One or more labels"
-                   " not approved.")
+        if self.is_forced():
+            if not self.is_release_approved(approvals):
+                msg = "Could not send to upstream: Release not approved."
+                logger.debug("Change %s: %s" % (self.change_id, msg))
+                ssh.exec_once('gerrit review -m %s %s'
+                              % (pipes.quote(msg), self.revision))
+                return
+            msg = ("***WARNING***\n%s (%s) has requested a forced upstream "
+                   "push. Bypassing all votes except for Release...\n\n"
```

Comment:
akerr: tclayton does not give me a release +1 without my CR, QA and OSRB approval...Also, this line is just logging:
```
***WARNING***
John Doe (john.doe@whocares.com) has requested a forced upstream push. Bypassing all votes except for Release...
```
When, this action is actually triggered by a comment from John Doe. John Doe cannot delete his comment on gerrit. He doesn't have access to it. You can print the votes at the point and gain your audit trail.

How about this:
```
Forcing upstream push. Votes at this point: 
...
```

I understand that we cannot implement a synchronous confirmation type action. One amazing thing about NetApp and our team is that we trust and help each other to do the right thing.. We don't go hunting for checking whose hands are stained when all they are doing is probably meeting deadlines and helping us by going the extra-mile.